### ISSUE TITLE
Deep learning model evaluation

### ISSUE THREAD

**Issue by melicoretta** [MarcSaghir/OCRReceiptProject#1]
Evaluate deep learning model and save the model of the locale storage for prediction

1) Confusion Matrix for Multi-Class Classification.
2) Calculating the Confusion Matrix with Scikit-learn.
3) Accuracy, Precision, and Recall.
4) Precision or Recall?
5) Conclusion.

**Comment by abhijeetGithu** [comment]
1. **Confusion Matrix for Multi-Class Classification.**
```python
from sklearn.metrics import confusion_matrix
import numpy as np

# Assuming you have the true labels and predicted labels
true_labels = [2, 0, 1, 2, 0, 1, 2]
predicted_labels = [2, 0, 2, 1, 0, 2, 2]

# Get the confusion matrix
cm = confusion_matrix(true_labels, predicted_labels)

# Print the confusion matrix
print(cm)

``` 
2. **Accuracy, Precision, and Recall**.
 So how to calculate Precision Accuracy Recall here is the code ;-
These metrics can be computed using scikit-learn's **classification_report** function. Here's an example:
```python
from sklearn.metrics import classification_report
# let assume the true labels with predicted labels
true_labels = [2, 0, 1, 2, 0, 1, 2]
predicted_labels = [2, 0, 2, 1, 0, 2, 2]

report = classification_report(true_labels, predicted_labels)
print(report)

```
